src/IssueViz.Security/HotspotsList/HotspotsControl.xaml

```diff
@@ -240,7 +240,8 @@
                   IsSynchronizedWithCurrentItem="True"
                   SelectedItem="{Binding SelectedHotspot, Mode=TwoWay}"
                   ItemsSource="{Binding Hotspots}"
-                  vsTheming:ImageThemingUtilities.ImageBackgroundColor="{Binding Background, RelativeSource={RelativeSource Self}, Converter={StaticResource BrushToColorConverter}}">
+                  vsTheming:ImageThemingUtilities.ImageBackgroundColor="{Binding Background, RelativeSource={RelativeSource Self}, Converter={StaticResource BrushToColorConverter}}"
+                  HorizontalScrollBarVisibility="Auto">
```

Comment:
It's possible to manually resize the columns and tool window so the columns aren't all visible, in which case we'll want a scrollbar to appear.